samples/msal-node-samples/EndToEndTestApp/app/test/testUtils.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Screenshot } from "../../../../e2eTestUtils/TestUtils";
+import { Page } from "puppeteer";
+import fs from "fs";
+
+// Constants
+export const SCREENSHOT_BASE_FOLDER_NAME = `${__dirname}/screenshots`;
+export const SAMPLE_HOME_URL = "http://localhost:3000";
+export const SUCCESSFUL_GRAPH_CALL_ID = "graph-called-successfully";
+export const SUCCESSFUL_GET_ALL_ACCOUNTS_ID = "accounts-retrieved-successfully";
+
+export async function enterCredentials(page: Page, screenshot: Screenshot, username: string, accountPwd: string): Promise<void> {
+    await page.waitForSelector("#i0116");
+    await screenshot.takeScreenshot(page, "loginPage");
+    await page.type("#i0116", username);
+    await screenshot.takeScreenshot(page, "loginPageUsernameFilled")
+    await page.click("#idSIButton9");
+    await page.waitForSelector("#idA_PWD_ForgotPassword");
+    await page.waitForSelector("#idSIButton9");
+    await screenshot.takeScreenshot(page, "pwdInputPage");
+    await page.type("#i0118", accountPwd);
+    await screenshot.takeScreenshot(page, "loginPagePasswordFilled")
+    await page.click("#idSIButton9");
```

Comment:
May want to add "keep me signed in" logic browser tests implement, just in case.

Reply:
Will do

Reply:
Added!